__init__.py

```diff
@@ -211,7 +210,8 @@ def on_websettings_changed(self):
                     self._register_all_intents()
                     self._is_setup = True
             except Exception as e:
-                self.log.error(e)
+                self.log.error("websettings_changed():threw %s" % (e,))
```

Comment:
using `self.log.exception` will simplify this.  it automatically prints the traceback